### ISSUE TITLE
[ramuda] Configuration change should also deploy lambda

### ISSUE THREAD

**Issue by saksham** [glomex/gcdt#93]
When we run `ramuda deploy` without changing code, but after changing the conf files, ramuda should trigger redeployment. `Currently it says Code hasn't changed - won't upload code bundle`. If the updated config is uploaded, the message is misleading. However, if we are not uploading the updated lambda, we need to do so.

**Comment by jbrandstetter** [comment]
Improve the message so it's clear what happened

**Comment by markfink** [comment]
extended the info message:
```
AWS Lambda code hasn't changed - won't upload code bundle
Update AWS Lambda configuration for function: infra-dev-sample-lambda-unittest
```

**Comment by markfink** [comment]
fixed with #145
